### DISCUSSION TITLE
Great plugin

### DISCUSSION THREAD

**Discussion by LozmindaSnr** [timbrel/GitSavvy#1801] [General]
I just wanted to say this is a great plugin. Thank you. Lozminda 🐰🔪

**Comment by LozmindaSnr** [comment]
**As an addendum (having read a few of the pull requests/discussions):**
Can I ask a really dumb question ? I'm assuming that further mods/dropping support for ST3 won't break the existing plugin for ST3?
(Unfortunately the ST4 roll out occurred just as I was getting grips with ST3, I accidentally upgraded to ST4 and it broke everything [(see the forum, one of the shorter threads](https://forum.sublimetext.com/t/st3-update-now-is-st4/57892/117)) and I prefer the look and lay out of ST3 (they changed all the default themes, a quite few behaviors etc etc)
I'm guessing changes that are being made for ST4 won't affect backward compatibility ? If I can (time is my issue) help, I'm happy to.
Thanks, Lozminda 🐰🔪

**Reply by kaste** [reply]
Yeah, hi and thanks.  The ST3 version won't break anytime but doesn't get any further features.  It is basically frozen just like ST3 itself.   But the ST3 version is pretty good.  Next versions of GitSavvy will probably get support for worktrees.  I would like to see that at least.  I think the commit view got better since the last ST3 version which I would miss.

**Reply by LozmindaSnr** [reply]
Hi again.
Thanks by the way for doing this in English, much appreciated.

Am thinking about making a financial contribution as I'm finally earning a small amount as a part time dev and GitSavvy has helped with my learning. It says that a donation will help the project, I'm quite happy for DeathAxe to get some cash too, as he's helped with no end with my endless ST3 questions (I bought a license a few months ago), how would a contribution be spent ? I.e. is it shared amongst the contributors ? T.B.H. I'm happy for it go to the dev's beer fund (or whatever) devs get thirsty too! It'd only be $30 or so (~ euros 30) maybe a little more..

All the best,

Lozminda 🐰🔪

**Reply by kaste** [reply]
The sponsor link is currently https://paypal.me/herrkaste 

It is also linked on the main page https://github.com/timbrel/GitSavvy

**Reply by LozmindaSnr** [reply]
Hi
Very sorry about the delay, it's only a small amount, but I'm having banking issues, my credit card linked to paypal has died unexpectedly. It _**is**_ the banks problem not mine, which helps neither of us. I will make that payment, only death, bad injury or extreme weirdness will stop me, it is probably going to take me a week or two more, hopefully less. Again apologies. 
All the best
Loz
